testing/kill_test.sh

```diff
@@ -0,0 +1,6 @@
+# script to test with specific verbosity and parameters
```

Comment:
Do you still need this file ? I'm not sure what is the use of it.

Reply:
Good point, I've renamed the file and added some explanations to make it more generically usable, I think it could be useful again in the future.

Reply:
Great and your changes seams to be working. Yeah !